packages/web-twig/src/Resources/components/TextFieldBase/README.md

```diff
@@ -65,6 +65,7 @@ Without lexer:
 | `helperText`        | `string`                                                             | `null`  | no       | Custom helper text                                                                                                                 |
 | `UNSAFE_helperText` | `string`                                                             | `null`  | no       | Unescaped custom helper text                                                                                                       |
 | `id`                | `string`                                                             | —       | yes      | Input and label identification                                                                                                     |
+| `inputProps`        | `string[]`                                                           | `[]`    | no       | Pass additional attributes for the input/textarea element                                                                          |
```

Comment:
```suggestion
| `inputProps`        | `string[]`                                                           | `[]`    | no       | Pass additional attributes to the input/textarea element                                                                          |
```

This sound better to me (and to ai assistant 😁 )

Reply:
Also please if you decide to change it, do it everywhere.

Reply:
Ok, I changed this on places where I add this description.